webcam_runner.py

```diff
@@ -0,0 +1,28 @@
+import vision
+import logging
+import cv2
+
+
+def main() -> None:
```

Comment:
It was mentioned that scripts should possibly exist in some kind of examples or testing folder. This looks like a script that will only be run rarely to ensure video streaming is working correctly, and if that is the case, I don't think this belongs in the main directory.

Reply:
Good point.